### PR TITLE
Add Fresh Start 2021 sale banner

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR implements the Fresh Start 2021 sale banner.

Fixes 1196341175636977-as-1199527862646240

### Implementation notes

- Removed New Year 2021 sale banner
- Added Fresh Start 2021 sale banner instead (campaign run from Feb 1 through Feb 14)
- Banner has no translation at the moment
- No experiment has been implemented at the moment

### Testing instructions

- Download the PR
- Run both Calypso and cloud locally
- Open your console
- Check that you see the new banner (both on mobile and desktop) in the Calypso plans (`/plans/:site`) and connect page (`/jetpack/connect/store`), and in the cloud pricing page (`/pricing`)
- Select another language than English in your profile
- Check that you still see the banner, in English
- Add the query parameter and value `?freshstart=false`, and verify that you don't see the banner
- Verify that there's no error in the console

### Screenshots

_Cloud mobile_
Screen Shot 2021-02-01 at 3 23 53 PM

_Cloud desktop_
Screen Shot 2021-02-01 at 3 23 44 PM

### CODE REVIEW

File: client/components/jetpack/fresh-start-2021-sale-banner/index.tsx [hunk 1]
```diff
@@ -31,15 +31,15 @@ type TimeWindowProps = {
...
-		// Start at midnight UTC on Jan 1,
-		// and continue until 23:59:59 UTC on Jan 18
-		const start = moment.utc( '2021-01-01' );
-		const end = moment.utc( '2021-01-19' );
+
+	if ( ! paramValue ) {
+		// Start at midnight UTC on Feb 1,
+		// and continue until 23:59:59 UTC on Feb 14
+		const start = moment.utc( '2021-02-01' );
+		const end = moment.utc( '2021-02-14' );
```
Comment (mattgawarecki) [inline client/components/jetpack/fresh-start-2021-sale-banner/index.tsx]: If I'm reading this correctly, the promo banner will stop being visible as soon as the system clock reads 2021-01-14, meaning the moment directly after 23:59:59 on February 13. If I understood correctly, we want this to be visible thru the last moment of February 14, so `end` should be like so:

```suggestion
		const end = moment.utc( '2021-02-15' );
```

Comment (monsieur-z) [reply client/components/jetpack/fresh-start-2021-sale-banner/index.tsx]: Good catch, thanks!

**Review by mattgawarecki — APPROVED** [review]
Everything tested well for me. Looks good, let's ship it! 🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5487177

Thank you @monsieur-z for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5487177).

**Pull request merged**